src/Providers/ViewServiceProvider.php

```diff
@@ -95,4 +127,19 @@ public function boot(): void
             }
         });
     }
+
+    public static function getRealPackageAssetPath(string $path, string $packageName): string
+    {
+        $path = ltrim($path, '/');
+        $relativePath = ltrim(
+            substr(
+                realpath(InstalledVersions::getInstallPath($packageName)),
+                strlen(realpath(array_keys(ClassLoader::getRegisteredLoaders())[0] . '/../')),
+                999
```

Comment:
why set length to 999?
`
If length is omitted or null, the substring starting from offset until the end of the string will be returned.
`